docs/VnVPlan/VnVPlan.tex

```diff
@@ -549,8 +549,318 @@ \subsection{Tests for Nonfunctional Requirements}
 be involved? Be specific.}
 
 \subsubsection{Look and Feel Requirements}
+These tests are to ensure that the way the front end is designed adheres to the
+defined requirements with respect to layout, typography, component alignment,
+etc. 
+
+\begin{description}
+  \item[Front End Component Review:] We will engage a developer to look through
+  the code and interface and ensure components are implemented as they are
+  defined in the SRS.
+  \item[Accessibility Check:] A developer will review if accessibility tools
+  exist or not within the application. Important tools are screen readers and
+  alt text for images. Furthermore, the developer must check for sufficient
+  colour contrast between components.
+\end{description}
+
+
+\begin{enumerate}
+
+\item{NFR-UI1}
+
+NFR: LF-AR1, LF-AR2, LF-AR3, LF-SR1, LF-SR2, LF-SR3, LF-SR4
+
+Type: Manual, Front End Review
+					
+Initial State: Application on home page, ready for use
+					
+Input/Condition: Tester engagement
+					
+Output/Result: List of all identified inconsistencies that are found that do not adhere to SRS defined requirements.
```

Comment:
Can we list them explicitly? I.e. For each NFR we should mention what we should be adhering to.

Reply:
Addressed in 91e5c568de4fc8efc5f61fffd99b982b9db10ebc